examples/demo-stf/src/genesis_config.rs

```diff
@@ -0,0 +1,69 @@
+/// Creates config for a rollup with some default settings, the config is used in demos and tests.
+use crate::runtime::GenesisConfig;
+use election::ElectionConfig;
+pub use sov_modules_api::default_context::DefaultContext;
+use sov_modules_api::default_signature::private_key::DefaultPrivateKey;
+use sov_modules_api::Context;
+use sov_modules_api::Hasher;
+use sov_modules_api::PublicKey;
+use sov_modules_api::Spec;
+pub use sov_state::config::Config as StorageConfig;
+use value_setter::ValueSetterConfig;
+
+pub const TEST_SEQUENCER_DA_ADDRESS: [u8; 32] = [1; 32];
+pub const LOCKED_AMOUNT: u64 = 200;
+pub const TEST_SEQ_PUB_KEY_STR: &str = "seq_pub_key";
+pub const TEST_TOKEN_NAME: &str = "sov-test-token";
```

Comment:
nit: it looks like we're using variables named `TEST_` for things other than testing (we export this demo config). Should we rename?